pkgs/development/libraries/libcamera/default.nix

```diff
@@ -65,7 +81,11 @@ stdenv.mkDerivation {
     openssl
   ];
 
-  mesonFlags = [ "-Dv4l2=true" "-Dqcam=disabled" ];
+  mesonFlags = [
+    "-Dv4l2=${mesonBool true}"
```

Comment:
Why add functions when the values are just static?

Reply:
Future proofing mostly (for making the features accessible as parameters to the derivation), and to make the types of the options more explicit.

Reply:
I would not say it does a very good job in either – it adds clutter to the expression that might (or might not) be used in the future and actually obscures the value that is passed to the options. And the type that accepts `enabled`/`disabled` is called `feature`.

Reply:
Fair enough, I'll just remove it for now. And rename the one in the pipewire expression (where I copied this from) to mesonFeature.